# Welcome to the event page of the next evolVienna meeting on 27<sup>th</sup> October!

[Schedule](#schedule) | [Sign-up](#sign-up) | [Posters](#poster-session) | [Support & contact](#support-and-contact) | [Tutorial 1](#tutorial-1) | [Tutorial 2](#tutorial-2) | [Tutorial 3](#tutorial-3)

<!-- [![evolVienna logo](https://www.evolvienna.at/mediatypes/srcsetvar_medium/evolvienna.jpg)](https://www.evolvienna.at/) -->

[![evolVienna meeting invite poster](evolVienna_meeting_27_Oct_2020_invite_poster_low_quality.jpg)](evolVienna_meeting_27_Oct_2020_invite_poster.pdf)

Due to COVID-19, this [evolVienna](https://www.evolvienna.at/) meeting will take place __online__. We will use the video conferencing system [Zoom](https://zoom.us/) - __no__ paid account is required. We will have __six__ interesting [speakers](#schedule) and we are delighted to announce that this meeting will also include an online [poster session](#poster-session)!

> Registration for this event closes at 13<sup>th</sup> October.

Please make sure that you are able to enter Zoom meetings using a __Meeting ID__ (not an invite link).

## Schedule

The meeting will take place __online__ on Tuesday, 27<sup>th</sup> October, 2020, between 13:00 and 17:00 (CEST).

All talks besides the poster session will take place in the same Zoom meeting room (__main room__) provided by _IST Austria_. We will send the details of the meeting room and password closer to the date.

After the first block of talks (14:40), you can leave the _main room_ in order to visit different _poster rooms_ during the poster session (14:45-15:25). Please make sure that you arrive back in the _main room_ before the second block of talks starts (15:30).

The following schedule is preliminary:

Start time (CEST) | End time (CEST) | Presenter               | Title                          | Zoom _Meeting ID_
:----------|:---------|:------------------------|:-------------------------------|:---------
13:00 | 13:10 | Organizing committee | Introduction | TBA
13:10 | 13:40 | Andreas Bergthaler (CeMM Vienna) | _Hidden tales about intra-host diversity, transmission bottlenecks and super-spreading events in the SARS-CoV-2 genome_ | TBA
13:40 | 14:10 | Tal Dahan/Tom Ellis (GMI & Weizman Institute) | _Resilience of a wild wheat population structure over 36 years of collection_ | TBA
14:10 | 14:40 | Nayuta Yamashita (University of Veterinary Medicine Vienna)| _Feeding ecology of dry forest lemurs_ | TBA
14:45 | 15:25 | Break and [poster session](#poster-session) |    |    
15:30 | 16:00 | Alevtina Koreshova (IMBA Vienna) | _Selfish toxin-antidote elements in animals: discovery, mechanisms and implications for speciation_ | TBA
16:00 | 16:30 | Sheng-Kai Hsu (University of Veterinary Medicine Vienna) | _Evolution of reproductive isolation during adaptation in allopatric populations with shared ancestry_ | TBA
16:30 | 17:00 | Réka Kelemen (IST Austria) | _Evolutionary and functional genomics of the t-haplotype, a model meiotic driver_ | TBA

## Sign-up

Please sign-up [here](https://forms.gle/PtZAfgpKZUjDpU3Z9) if you are planning to attend the meeting as __audience__.

If you want to present a __poster__ please sign-up [here](https://forms.gle/octdQ5fc9dj52EuC7).

Both registrations __close at October 13<sup>th</sup>__. Shortly after this day, the _audience_ and _poster presenters_ will receive the remaining necessary information via email from one of the members of the organizing committee.

## Poster session

The idea is that presenters upload their poster to [figshare](https://figshare.com), and links to each poster will be posted here. Each presenter will host a (free, 40-minute) Zoom meeting where they can discuss it, and the audience can freely enter and leave meetings as they please. Since we are not limited by physical space, we can __accept all poster submissions__, so we encourage as many folks as possible to present.

We will send detailled instructions about setting up figshare and your Zoom meeting after registration has closed on 13th October by email. The Zoom discussions will be private to people who have registered for the meeting. However, please be aware that posters shared on figshare will be public. If you do not want your poster to be published online, please contact Ben Wölfl (benjamin.woelfl@univie.ac.at). 

The Zoom rooms indicated by the Zoom _Meeting IDs_ below will only be active on October 27<sup>th</sup> between __14:45 and 15:25__ (CEST). The required password for all rooms is the same and will be sent out via email to all attendees.

> You will see a gradual accumulation of the actual posters here as they rush in.

---

#### Poster 01: Maintenance Mechanisms of Trans-Specific Polymorphism in the _Arabidopsis_ Clade

Authors: __Krzysztof Stankiewicz__<sup>1</sup> et al.

Affiliations: <sup>1</sup> Department of Mathematics, University of Vienna & Vienna Grad. School of Pop. Gen.

Zoom _Meeting ID_: TBA

TBA

---

#### Poster 02: TBA

---

#### Poster 03: Multiple haplotype reconstruction from allele frequency data

Authors: __Marta Pelizzola__ <sup>1</sup>, Merle Behr, Housen Li, Axel Munk, Andreas Futschik

Affiliations: <sup>1</sup> Department of Population Genetics, Veterinary Medical University of Vienna & Vienna Grad. School of Pop. Gen.

Zoom _Meeting ID_: 799 9746 3341

Evolve and Resequence (E&R) methods follow many replicate populations of a given model organism, like _C. elegans_ or _Drosophila_, throughout time in order to understand how it adapts to different conditions. It has been shown that performing pool sequencing of one population instead of individual sequencing can give the same sequencing coverage with much lower costs. Furthermore, the power of detecting SNPs being targets of selection stays high. However, by doing so, the haplotype information is lost. Starting from pool sequenced data, our goal is to characterize the haplotype structure and to disentangle the possible patterns of selection in the data. We are investigating trajectories of haplotypes given different selection pressures, how the haplotype structure can help in identifying the truly causal SNPs and which starting condition of the experiment would favor the haplotype reconstruction and the detection of the SNPs driving adaptation. 
The first step of our research consists in reconstructing the haplotype structure and the haplotype frequency given time series of allele frequency from pool sequencing only. We are currently aiming to extend this work and we are working on a method to detect signatures of selection using the haplotype information we are able to retrieve from the allele frequencies.

<iframe src="https://widgets.figshare.com/articles/13110689/embed?show_title=true" width="100%" height="250px" allowfullscreen frameborder="0"></iframe>

---

#### Poster 04: Temperature adaptation in _Arabidopsis thaliana_

Authors: __Pieter Clauw__ <sup>1</sup>, Magnus Nordborg

Affiliations: <sup>1</sup> Vienna Bio Center

Zoom _Meeting ID_: 743 3769 8349

Climate variation exerts significant evolutionary pressure on plants. _Arabidopsis thaliana_ has a large geographic range and local varieties have acquired specific adaptations to their local environment. As a winter-annual, A. thaliana grows during autumn under decreasing temperatures that are usually considerably lower than standard lab conditions. Northern varieties may never experience temperatures above 10ºC until they are about to flower in spring. Although we expect northern strains (natural inbred lines, or “accessions”) to be adapted to lower temperatures, the importance of temperature adaptation and the mechanisms involved are not known.
We exposed 250 accessions from south-Spain to northern-Sweden to ecological relevant autumn temperatures. Our experiments indicate that growth of accessions coming from colder regions is less affected by colder temperatures compared to accessions from warmer regions.
Genome wide association mapping (GWAS) hints towards a polygenic trait architecture.
Besides growth measurements we have obtained primary metabolome data for 241 accessions in both temperatures. The over all 37 metabolites we find the northern accessions to show a much more pronounced metabolic response to the temperature difference than the southern accessions. We think the increased metabolic response is an actively regulated preparation for colder temperatures, a so-called cold acclimation. The increased expression of a set of well-described cold acclimation genes in a limited set of northern accessions seems to confirm that idea.
Along those lines it is also to be expected that the smaller rosette sizes in northern accessions are due to an active growth inhibition. Smaller plants cope better with adverse conditions.
We are now further investigating the polygenicity of plant growth and it's response to temperature by running GWAS and calculating polygenic scores. To reduce the complexity of plant growth we are also attempting to split up this complex trait in less-complex metabolic sub-traits. Together with the hypothesis that cold acclimation is playing an important role in temperature adaptation we hope to further elucidate one aspect of local adaptation in _Arabidopsis thaliana_.

---

#### Poster 05: A novel toxin-antidote element in _C. tropicalis_

Authors: __Daniel Krogull__<sup>1</sup>, Polina Tikanova

Affiliations: <sup>1</sup> Vienna Bio Center

Zoom _Meeting ID_: 831 7991 6389

<iframe src="https://widgets.figshare.com/articles/13118633/embed?show_title=true" width="100%" height="250px" allowfullscreen frameborder="0"></iframe>

---

#### Poster 06: Illuminating the genetic basis of the Picea-Ips-Pathosystem. A Pilot Project

Authors: Kelly Swarts, Miroslav Polacek, Alexis Arizpe, __Lisa Weidlich__ <sup>1</sup>

Affiliations: <sup>1</sup> Vienna Bio Center

Zoom _Meeting ID_: 865 2892 9890

In the northern hemisphere the European Spruce Bark Beetle (Ips typographus) is one of the major disturbance factors in Norway Spruce (Picea abies) stands and the severity is expected to increase with climate change. To understand the genetics underlaying the Picea-Ips pathosystem dynamics better, we established our first pilot project, including repeated paired sampling of both spruce and beetles from plots under immediate threat of bark beetle infestation in central Europe. Our goals are (1) to understand genetic consequences of epidemic phase bark beetle population dynamics over the course of a year, (2) to identify regions of the genome associated with virulence in beetles and resistance in trees and (3) to test for selective response to the partner phenotype.

<iframe src="https://widgets.figshare.com/articles/13118465/embed?show_title=true" width="100%" height="250px" allowfullscreen frameborder="0"></iframe>

---

#### Poster 07: Genetic architecture for CG gene-body methylation in _Arabidopsis thaliana_

Authors: __Rahul Pisupati__ <sup>1</sup>, Magnus Nordborg

Affiliations: <sup>1</sup> Vienna Bio Center & Vienna Grad. School of Population Genetics

Zoom _Meeting ID_: 793 9918 3203

Transgenerational epigenetic inheritance is a common phenomenon in plants. Experiments in Arabidopsis thaliana show that epigenetic marks, such as cytosine methylation, can be faithfully inherited over generations. This type of inherited material is often termed the epi-genotype. However, recent genome-wide association studies (GWAS) using epigenetic marks as traits have shown that there are major-effect loci affecting methylation across the genome. These findings suggest that the variation we observe is instead a phenotype. In addition, transposable elements (TEs) and repetitive elements would change methylation in their proximity. It remains unclear how the above factors shape the global pattern of methylation variation. The goal of the project is to quantitatively distinguish various factors affecting methylation variation in natural lines. Here, I am mainly focusing on CG gene-body methylation variation since it is shown to be faithfully inherited and determine its genetic basis.

---

#### Poster 08: Three-way crosses of _Drosophila melanogaster_ inbred strains provides insight into epistatic interactions and dominance relationships using Evolve and Resequencing 

Authors: __Claire Burny__ <sup>1</sup>, Viola Nolte, Marlies Dolezal, Christian Schlötterer

Affiliations: <sup>1</sup> Veterinary Medical University of Vienna & Vienna Grad. School of Pop. Gen.

Zoom _Meeting ID_: TBA

Many adaptive traits are determined by numerous genes and the smaller the genes contributions, the more additivity is expected. Consequently, polygenicity complicates the study of genetic interactions as weak epistasis might be common for highly polygenic traits but hard to quantify. Experimental evolution is a useful approach to study the adaptive architecture, using replicated populations maintained in new environments. We propose to study the interplay between effect size, the degree of additivity and dominance with an Evolve and Re-sequencing (E&R) design using three Drosophila melanogaster inbred strains - Samarkand (S), Oregon-R (O) and Helsinki (H). Four replicates of two-genotype combinatorial crosses (S⨉O, S⨉H and O⨉H) are maintained in an extreme thermal environment (constant 29°C). All four replicates display a strong and parallel selection response within each cross after 10 generations. However, the frequency of a given genotype is dependent on the cross; this suggests that pervasive (high-order gene-by-gene) epistatic interactions can be detected in our evolved populations that experienced strong allele frequency changes. We then use the allele frequency of a given genotype across the crosses to refine the selection coefficient (s) and dominance (h) estimates and to test different assumptions about dominance (additivity, deviation from additivity and possibly s vs h dependencies). We plan to use the relative allele frequency across the crosses to infer dominance hierarchy between the three genotypes. We eventually suggest that such replicated E&R combinatorial crosses allow to reveal latent epistasis and to refine the estimates of dominance.

---

#### Poster 09: Chromatin conformation in _Drosophila melanogaster_ dosage compensated male X-chromosome

Authors: __Lubna Younas__<sup>1</sup>, Mujahid Ali, Qi Zhou

Affiliations: <sup>1</sup> Althanstraße, University of Vienna

Zoom _Meeting ID_: TBA

In _Drosophila melanogaster_ the single male chromosome X (chrX) undergoes
an average twofold transcriptional upregulation for balancing the
transcriptional output between sexes. All the previous dosage compensation
(DC) studies focused on linear information instead of 3D chromatin
conformation. In our study, we found that DC in Drosophila is spatiotemporally
regulated and the degree of DC varies during development. Then further we
explored the role of chromatin 3D organization in binding and localization of
MSL-complex. we examined the 3D chromatin organization of dosage
compensated chrX using high-resolution Hi-C data in different developmental
stages and adult tissues. We detected the global and local differences in the
long-range interaction profiles in dosage compensated male chrX. We also
noticed that the male chrX has more long-range interactions that increases
during development and realized that the single copy male chrX has a more
flexible and open chromatin. Our observations indicated that MSL binding sites
(HAS and PionX) are in spatial contacts located within active chromatin regions
on the chrX, while PionX sites showed higher long-range interactions during
the early establishment of DC and later replaced by HAS. However, the 1.688x 3F
satellite repeats play an important role in recognition of chrX and facilitate the
binding and spreading of MSL-complex during development by modulating the
local chromatin environment through long range interactions.
Together, our data indicate that chromosome X dosage compensation affects
global and local chromosome structure.

---

#### Poster 10: _In vitro_ reconstruction of avian skin and comparative transcriptomics suggest evolutionarily ancient control of epidermal barrier functions in terrestrial vertebrates

Authors: __Julia Lachner__ <sup>1</sup>, Sophia Derdak, Florian Ehrlich, Tanja Wagner, Michael Mildner, Erwin Tschachler, Leopold Eckhart

Affiliations: <sup>1</sup> Medical University of Vienna

Zoom _Meeting ID_: TBA

The evolutionary transition to life on land depended on an efficient skin barrier
against mechanical insults and transcutaneous water loss. This barrier is
formed and constantly renewed by epidermal keratinocytes which differentiate
into rigid corneocytes and secrete lipids to seal the intercellular space on the
skin surface. We hypothesized that the epidermal barrier function of terrestrial
vertebrates is controlled by an evolutionarily ancient gene expression program
shared by phylogenetically diverse amniotes. To test this hypothesis, we
determined the dynamics of global gene expression during avian keratinocyte
differentiation in a newly established 3-dimensional in vitro skin model and
compared it with a mammalian counterpart (human skin model). The common
differentiation program of chicken and human keratinocytes included keratin
intermediate filaments, proteins of the epidermal differentiation complex (EDC),
enzymes involved in the synthesis of skin barrier lipids and a distinct set of
other proteins. Interestingly, interleukin-36 receptor antagonist (IL36RN) was
transcriptionally upregulated in differentiating keratinocytes of both species,
suggesting that this anti-inflammatory protein plays a conserved role in the
suppression of inflammation at the skin barrier. Many but not all of the
terrestrial skin barrier genes are specific for amniotes and expressed only in
the epidermis. In conclusion, the comparative analysis of keratinocytes from
phylogenetically diverse species under defined in vitro differentiation
conditions identifies evolutionary ancient genes the control epidermal barrier
formation and maintenance.

---

#### Poster 11: Selective elimination of positive length variants contributes to genome shrinkage in _Arabidopsis_

Authors: __Robin Burns__<sup>1</sup>, Magnus Nordborg

Affiliations: <sup>1</sup> Vienna Bio Center

Zoom _Meeting ID_: TBA

Angiosperms (flowering plants) vary almost 2000 fold in their genome size. Major processes that lead to genome size expansion are: polyploidization, transposable element (TE) proliferation and supernumerary ribosome DNA (rDNA) copies. Countering this, several mechanisms leading to genome-shrinkage include: the wholesale loss of chromosomes, ectopic recombination and single strand annealing during DNA repair. While the molecular mechanisms underlying these processes are being studied, the nature of the relationship between selection and genome size is not well understood. Arabidopsis is a useful genus to study large genome size change over short evolutionary time. The selfing plant A. thaliana has a small streamlined genome (125Mb) compared to that of the outcrossing perennial A. lyrata (207Mb), which has a more repeat rich genome and ~200Mb is the average genome size for the rest of the genus. Previous work attributed the smaller genome size in A. thaliana to multiple small deletions occurring along the branch after the species split ~6 Mya. However, this conclusion was based on a single reference genome for both species. By assembling 2 additional long read reference genomes of A. lyrata we find many of these deletions to be segregating in A. lyrata as well (and largely fixed in A. thaliana). This suggests that strong selection against positive length variants from standing variation in the ancestor species contributed to genome shrinkage in A. thaliana, and that when comparing genome size, using a single reference genome for a species can lead to incorrect conclusions. Currently, we are examining short reads for a population of A. lyrata lines (n=125) using kmers to call the presence absence of these indels. We aim determine what selection is acting on them in A. lyrata and compare and contrast selection on genome size between selfers and outcrossers.

---

#### Poster 12: Transposon dynamics in _Arabidopsis thaliana_

Authors: __Luz Mayela Soto-Jimenez__ <sup>1</sup>, Magnus Nordborg

Affiliations: <sup>1</sup> Vienna Bio Center

Zoom _Meeting ID_: TBA

TBA

---

## Tutorials

---

### Tutorial 1

> How to __access__ a Zoom meeting as __audience__

Click [here](tutorial_zoom_attendee.pdf) to download the PDF with the tutorial.

---

### Tutorial 2

> How to __schedule and open__ a Zoom meeting as a __poster presenter__

Click [here](tutorial_zoom_presenter.pdf) to download the PDF with the tutorial.

---

### Tutorial 3

> How to __publish__ your poster on __figshare__

Click [here](tutorial_figshare.pdf) to download the PDF with the tutorial.

__Reminder:__ If you follow this procedure you are effectively __publishing__ your poster. If you do not like that, please contact a member of the organizing committee.

---

### Support and contact

In case you have any questions or feedback please contact a member of the _evolVienna meeting organizing committee_:
- Benjamin Wölfl (Maths dept - benjamin.woelfl@univie.ac.at)
- Tom Ellis (GMI - thomas.ellis@gmi.oeaw.ac.at)
- Christina Hedderich (Botany dept - christina.hedderich@univie.ac.at)
- Sanjay Narayanaswamy (Althanstraße, Uni Wien - sathyasaisanjay@gmail.com)
- Daria Shipilina (IST - daria.shipilina@ist.ac.at)

### Author and acknowledgements

Author: Benjamin Wölfl, Department of Mathematics, Oskar-Morgenstern-Platz 1, 1090 Wien, Austria

Big thanks to the _evolVienna_ community for support, attending, talks, poster submissions and lively discussions.

An image by <a href="https://pixabay.com/users/dianaparkhouse-9868127/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=4551691">Diana Parkhouse</a> from <a href="https://pixabay.com/?utm_source=link-attribution&amp;utm_medium=referral&amp;utm_campaign=image&amp;utm_content=4551691">Pixabay</a> was used on this website. Other images are derived from <a href="https://www.canva.com/">Canva</a> on a _free for noncommerial use_ license. Attributions are not required for both sources.

The author is not responsible for contents in any of the linked URLs. If you do not like something about this website, please create a pull request or contact us.
